u8timeseries/tests/test_models.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import pandas as pd
+import numpy as np
+
+from ..timeseries import TimeSeries
+from u8timeseries import Prophet, KthValueAgoBaseline, ExponentialSmoothing, TimeSeries, Arima, AutoArima
+from u8timeseries import StandardRegressiveModel
+from u8timeseries.models.theta import Theta
+
+class ModelsTestCase(unittest.TestCase):
+    __test__ = True
+
+    # general prediction / training parameters
+    forecasting_horizon = 5
+    regression_window = 5
+
+    # dummy timeseries for autoregression forecasting and target for regression
+    times = pd.date_range('20000101', '20000130')
+    values = np.sin(range(len(times))) + np.array([2.0] * len(times))
+    ts: TimeSeries = TimeSeries.from_times_and_values(times, values)
+
+    # dummy feature timeseries for regression
+    feature_values = np.array(range(len(times))).astype(float)
+    feature_ts: TimeSeries = TimeSeries.from_times_and_values(times, feature_values)
```

Comment:
Not directly related, but I think it would be nice to have a collection of "utils" to easily create "dummy" TimeSeries which could be used as features. For example one could be the constant time series. Another could be a linearly increasing one, or one which 1's during public holidays and 0's else, etc.

Reply:
That makes sense! I will add this to Jira.